app/mailers/jobapps_mailer.rb

```diff
@@ -9,11 +9,11 @@ def application_denial(application_record)
     @application_record = application_record
     @user = application_record.user
     template = application_record.position.application_template
-    if template.present?
-      reply_to = template.email
-    else
-      reply_to = ''
-    end
+    reply_to = if template
```

Comment:
Could also do

```ruby
reply_to = template.try(:email)
```

unless the mailer has a problem with a nil address rather than a blank string.